presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -10,10 +10,11 @@ Kafka Connector
 Overview
 --------
 
-This connector allows the use of Apache Kafka topics as tables in Presto.
+This connector allows the use of `Apache Kafka <https://kafka.apache.org/>`_ 
+topics as tables in Presto.
 Each message is presented as a row in Presto.
 
-Topics can be live: rows will appear as data arrives and disappear as
+Topics can be live, rows appear as data arrives and disappear as
```

Comment:
Shouldn't this be a period or a semicolon instead of a comma?

Reply:
Yeah ... shorter separate sentences are even better. Done.